lib/proc.c

```diff
@@ -3835,11 +3838,30 @@ selector_in_values_sequential_search(grn_ctx *ctx,
      *  grn_in_values_too_many_index_match_ratio)
     */
     if (n_existing_records >
-        (n_indexed_records * grn_in_values_too_many_index_match_ratio)) {
+        (n_indexed_records * too_many_index_match_ratio)) {
+      GRN_LOG(ctx,
+              GRN_LOG_INFO,
+              "%s[use-index] n_index_match_records:%d, "
+              "n_existing_records:%d, "
+              "too_many_index_match_ratio:%.2f",
+              tag,
+              n_indexed_records,
+              n_existing_records,
+              too_many_index_match_ratio);
```

Comment:
This is different with our existing report. See `select_index_use_sequential_search()` in `lib/table_selecctor.c` and `lib/report.c`.